packages/@aws-cdk/aws-elasticloadbalancingv2/README.md

```diff
@@ -95,6 +95,27 @@ both. If both are specified, both conditions must be met for the requests to
 be routed to the given target. `priority` is a required field when you add
 targets with conditions. The lowest number wins.
 
+Or you can add listner rule condition using `ApplicationListenerRule` directly:
+
+```
+const targetGroup = new ApplicationTargetGroup(this, 'TargetGroup', {
+  port: 8080,
+  protocol: ApplicationProtocol.HTTP,
+  targetGroupName: 'backend',
+  targets: [asg],
+  vpc: vpc,
+});
+
+const listenerRule = new ApplicationListenerRule(this, 'ListenerRule', {
+  listener,
+  priority: 10,
+  pathPatterns: ['/ok', '/path'],
```

Comment:
I think I'd like to see these fields `@deprecated` and a field added:

```
conditions?: Condition[];
```

(Obviously the @deprecated properties' code with go through the same code path).

Do this both for `ApplicationListenerRule` as well as for `listener.addTargets()` and `listener.addTargetGroups()`.